app/src/main/res/values/strings.xml

```diff
@@ -810,16 +810,16 @@
     <string name="security">Security</string>
     <string name="set_mode_for_app_ops_dots">Set mode for app ops…</string>
     <string name="backup_source_description">Back up the entire source directory.</string>
-    <string name="backup_apk_only_description">Back up only the APK files from the source directory. <b>Source</b> must be selected for this to work.</string>
-    <string name="backup_data_description">Back up data folders. If selected, internal data folders will be backed up by default.</string>
-    <string name="backup_external_data_description">Back up external data folders. <b>Data</b> must be selected for this to work.</string>
+    <string name="backup_apk_only_description">Only back up the APK files from the source directory. (<b>Source</b> must be selected for this to work.)</string>
+    <string name="backup_data_description">Backs up data-, and internal data folders.</string>
+    <string name="backup_external_data_description">Back up external data folders. (<b>Data</b> must be selected for this to work.)</string>
     <string name="backup_obb_media_description">Back up OBB and media folders.</string>
     <string name="backup_exclude_cache_description">Do not back up cache, no_cache and no_backup folders.</string>
     <string name="backup_extras">Extras</string>
-    <string name="backup_extras_description">Back up app permissions, battery saving, data usage options, MagiskHide status, etc.</string>
+    <string name="backup_extras_description">Back up app permissions, battery saving and data usage options, MagiskHide status, etc.</string>
```

Comment:
Add SSAID too. I probably forgot to revise the string.

Reply:
"stauts, SSAID, etc."? What is SSAID?

Reply:
https://muntashirakon.github.io/AppManager/en/main.html#x1-30001.1

Reply:
Link didn't work.

Reply:
It worked at that time. New link: https://muntashirakon.github.io/AppManager/en/index.html#x1-40001.1